crypto/src/merkle/concurrent.rs

```diff
@@ -18,6 +18,8 @@ pub const MIN_CONCURRENT_LEAVES: usize = 1024;
 // PUBLIC FUNCTIONS
 // ================================================================================================
 
+/// Builds merkle nodes.
```

Comment:
I would write this as: "Returns internal nodes of a Merkle tree constructed from the provided leaves."